### ISSUE TITLE
[Dropdown Menu], [Main Menu] No ability to select any option from the dropdown using keyboard

### ISSUE THREAD

**Issue by NatalliaAlieva** [epam/UUI#877]
### Description
User cannot click the avatar icon with options in the Dropdown Menu example when he/she using keyboard.

### Steps to Reproduce
1. Open [DropdownMenu](https://uui.epam.com/documents?category=components&id=dropdownMenu&skin=UUI4_promo) component.
2. Using keyboard click on icon with avatar.

### Actual result
User cannot click on the avatar icon from the keyboard.

https://user-images.githubusercontent.com/57751283/209883176-2ca7ddee-c1e1-4fa1-8333-6fb241556809.mp4

### Expected result
User should be able to perform the same actions with both the mouse and the keyboard.

### Description
User cannot select any option from the Dropdown Menu when he/she using keyboard.

### Steps to Reproduce
1. Open [DropdownMenu](https://uui.epam.com/documents?category=components&id=dropdownMenu&skin=UUI4_promo) component.
2. Using keyboard select any option from dropdown menu.

### Actual result
User cannot get into the dropdown menu from the keyboard.

https://user-images.githubusercontent.com/57751283/207852552-4c051fcf-fec8-4f33-ae3e-3f22f96ac8e6.mp4

### Expected result
User should be able to perform the same actions with both the mouse and the keyboard.

### Description
User cannot select any tab from the dropdown "More" when he/she using keyboard.

### Steps to Reproduce
1. Open [MainMenu](https://uui.epam.com/documents?id=mainMenu&mode=doc&skin=UUI4_promo&category=components#responsive) component.
2. Using `Tab` navigate to "More" tab.
3. Press `Enter` to open the dropdawn.
4. Using the keyboard select any option from dropdown menu.

### Actual result
User cannot get into the dropdown menu from the keyboard.

https://user-images.githubusercontent.com/57751283/209883758-a8fd90e9-ce77-43b6-a536-7aff12dbfa16.mp4

### Expected result
User should be able to perform the same actions with both the mouse and the keyboard.

### Description
User cannot click the avatar and global menu icons in the Main Menu component when he/she using keyboard.

### Steps to Reproduce
1. Open [MainMenu](https://uui.epam.com/documents?id=mainMenu&mode=doc&skin=UUI4_promo&category=components#basic) component.
2. Using keyboard click on icon with avatar.
3. Using keyboard click on global menu icon.

### Actual result
User cannot click on the avatar OR global menu icon from the keyboard.

https://user-images.githubusercontent.com/57751283/209961041-2bb7a01e-1773-4bd8-85ec-38d6ecd5c145.mp4

### Expected result
User should be able to perform the same actions with both the mouse and the keyboard.

### Description
Incorrect focus order after opening  "Main menu" (hamburger) button, focus is not trapped in opened dialog

### Precondition
Screen Reader is **OFF** or **ON**

### Steps to Reproduce
1. Open [MainMenu](https://uui.epam.com/documents?category=components&id=mainMenu&mode=doc&skin=UUI4_promo) component.
2. Zoom in the page so that the Main Menu (hamburger) button is displayed.
3. Use down arrow key (since currently button is not reachable using `Tab` key) to navigate to Main menu (hamburger) button and press `Enter` key to expand it.
4. Use `Tab` key to navigate to the first item in opened navigation menu and observe.
5. Use `Tab` key to navigate through the interactive elements in opened navigation menu and observe.

### Actual Result
After activating Main Menu (hamburger) button and opening navigation menu, pressing `Tab` key doesn't  move focus to the first item in opened navigation menu. Focus shis not trapped inside opened navigation menu.

https://github.com/epam/UUI/assets/57751283/08559801-38c9-4144-bc28-fdb86a3cc546

### Expected Result
After activating Main Menu (hamburger) button and opening navigation menu, pressing `Tab` key should move focus to the first item in opened navigation menu. Focus should be trapped in opened navigation menu, when last item is focused, pressing `Tab` key should move focus to the "x" close button, next pressing TAB - to the first item.
**Failed WCAG Checkpoint:** 2.4.3 Focus Order (Level A)
https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html

**Comment by NatalliaAlieva** [comment]
Released in 5.1.3 ver.
